subsystem/kubeapi_test.go

```diff
@@ -2702,7 +2702,6 @@ var _ = Describe("[kube-api]cluster installation", func() {
 	})
 
 	It("deploy clusterDeployment and infraEnv and with an invalid NMState config YAML", func() {
```

Comment:
Looks like the expected message ([line 2730](https://github.com/openshift/assisted-service/pull/4866/files#diff-f2d3cebc87fae81285493762229b8e7ecac6ab614ea0d82950af55f303dcc3d5L2730)) needs modification for the new error message

From [CI test](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-service/4866/pull-ci-openshift-assisted-service-master-edge-subsystem-kubeapi-aws/1611059844654043136)
```
{ Failure /assisted-service/subsystem/kubeapi_test.go:2704
Timed out after 120.000s.
Expected
    <string>: Failed to create image: 1 error occurred:
    	* failed to validate network yaml for host 0, nmstate GenerateConfiguration failed, error: failed when generating the configuration {"foo":"bar"} with rc: 1, err_msg: Invalid json string: Unsupported keys found: ["foo"], err_kind: InvalidArgument
    
    
to contain substring
    <string>: nmstate generated an empty NetworkManager config file content
/assisted-service/subsystem/kubeapi_test.go:504}
```

Reply:
Thanks, @CrystalChun. I actually had this change ready and I forgot to include the file in git.
Fixed now 😃

Reply:
Nice!